### ISSUE TITLE
cannot handle files without metadata header

### ISSUE THREAD

**Issue by nstbayless** [nstbayless/flyweb-mp3#22]
**Steps taken:**
- created playlist
- uploaded a few .mp3 files
- uploaded a .wav file

**Result:**
```
routes/api.js:81
                throw err;
                ^

Error: Could not find metadata header
    at node_modules/musicmetadata/lib/id3v1.js:13:19
    at Stream.done (node_modules/musicmetadata/lib/common.js:31:5)
    at emitNone (events.js:91:20)
    at Stream.emit (events.js:185:7)
    at drain (node_modules/musicmetadata/node_modules/through/index.js:34:23)
    at Stream.stream.queue.stream.push (node_modules/musicmetadata/node_modules/through/index.js:45:5)
    at Stream.end (node_modules/musicmetadata/node_modules/through/index.js:15:35)
    at _end (node_modules/musicmetadata/node_modules/through/index.js:65:9)
    at Stream.stream.end (node_modules/musicmetadata/node_modules/through/index.js:74:5)
    at ReadStream.onend (_stream_readable.js:511:10)
```

Not sure if we're intending to handle .wav files, but at the very least we should fail gracefully. @sinoforever 

**Edit**: this might actually be caused by any file without metadata, because I tried uploading another file without metadata and it threw the same error. We should definitely accept files without metadata.

**Comment by sinoforever** [comment]
I'll look into it

**Comment by nstbayless** [comment]
By the way, have you been able to reproduce this? I can send you the files that caused the crash if you like.

**Comment by sinoforever** [comment]
Yea I actually found that during sprint, but haven't wrote the handler for it.

**Comment by sinoforever** [comment]
fixed in da053d6
